robot-walker-four/src/main/java/us/ihmc/robotWalkerFour/RobotWalkerFourController.java

```diff
@@ -301,6 +301,25 @@ public void doControl()
       }
    }
 
+   /**
+    * A {@code CenterOfMassCommand} is created using the new calculated {@code centerOfMassPosition}
+    * and adds the command to the controller core.
+    * 
+    * @param centerOfMassPosition refers to newly calculated position of the center of mass of the
+    *                             robot.
+    */
+   public void sendCenterOfMassCommand(FramePoint3D centerOfMassPosition)
```

Comment:
For the argument, use `FramePoint3DReadOnly`, it's essentially the same as `FramePoint3D` but indicates that the method will only read information from it by only requiring a read-only version.